test/test_checks.py

```diff
@@ -1,6 +1,18 @@
 from flake8_quotes import QuoteChecker
 import os
-from unittest import TestCase
+import subprocess
+from unittest import expectedFailure, TestCase
+
+
+FLAKE8_VERSION = int(os.environ.get('FLAKE8_VERSION', 2))
```

Comment:
Ehm no, you had a fallback which was an int! `FLAKE8_VERSION = os.env.get('FLAKE8_VERSION', 2)`